Kbs.Wpf/User/Update/AccountViewModel.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Kbs.Wpf.Components;
+
+namespace Kbs.Wpf.User.Update
+{
+    public class AccountViewModel : ViewModel
+    {
+        private string _inputEmail;
+        private string _inputPassword;
+        private string _inputConfirmPassword;
+
+        public string InputEmail { get => _inputEmail; set => _inputEmail = value; }
```

Comment:
Use SetField(ref _inputEmail, value):
```suggestion
        public string InputEmail { get => _inputEmail; set => SetField(ref _inputEmail, value); }
```